src/pages/NewPage.tsx

```diff
@@ -0,0 +1,164 @@
+  const intl = useIntl();
+  const lastSessionTitle = intl.formatMessage({
+    id: "newpage-last-session-title",
+  });
+  const continueSessionTitle = intl.formatMessage({
+    id: "newpage-continue-session-title",
+  });
+  const newSessionTitle = intl.formatMessage({
+    id: "newpage-new-session-title",
+  });
+
+  return (
+    <DefaultPageLayout
+      toolbarItemsRight={<HomeToolbarItem />}
+      menuItems={<HomeMenuItem />}
+    >
+      <LoadProjectInput ref={loadProjectRef} accept=".json,.hex" />
+      <VStack alignItems="center">
+        <Container maxW="1180px" alignItems="stretch" zIndex={1} p={4} mt={8}>
+          <VStack alignItems="stretch" w="100%">
+            <Heading as="h1" fontSize="4xl" fontWeight="bold">
+              <FormattedMessage id="newpage-title" />
+            </Heading>
+            <Heading as="h2" fontSize="2xl" mt={8}>
+              <FormattedMessage id="newpage-section-one-title" />
+            </Heading>
+            <HStack
+              w="100%"
+              gap={8}
+              alignItems="stretch"
+              mt={3}
+              flexDir={{ base: "column", lg: "row" }}
+            >
+              <NewPageChoice
+                onClick={handleOpenLastSession}
+                label={lastSessionTitle}
+                disabled={!existingSessionTimestamp}
+                icon={<Icon as={RiRestartLine} h={20} w={20} />}
+              >
+                {existingSessionTimestamp ? (
+                  <Stack mt="auto">
+                    <Text>
+                      <FormattedMessage
+                        id="newpage-last-session-name"
+                        values={{
+                          strong: (chunks: ReactNode) => (
+                            <Text as="span" fontWeight="bold">
+                              {chunks}
+                            </Text>
+                          ),
+                          name: projectName,
+                        }}
+                      />
+                    </Text>
+                    <Text>
+                      <FormattedMessage
+                        id="newpage-last-session-date"
+                        values={{
+                          strong: (chunks: ReactNode) => (
+                            <Text as="span" fontWeight="bold">
+                              {chunks}
+                            </Text>
+                          ),
+                          date: new Intl.DateTimeFormat(undefined, {
+                            dateStyle: "medium",
+                          }).format(existingSessionTimestamp),
+                        }}
+                      />
+                    </Text>
+                  </Stack>
+                ) : (
+                  <Text>
+                    <FormattedMessage id="newpage-last-session-none" />
+                  </Text>
+                )}
+              </NewPageChoice>
+              <NewPageChoice
+                onClick={handleContinueSessionFromFile}
```

Comment:
Opening a json data file doesn't seem to redirect the user to the data samples page at the moment for me. Opening a hex file seem to. Maybe it should?

Reply:
But this could be an existing issue because it seems like dragging a json file imports the data, but doesn't navigate to the data samples page in the react branch..

Reply:
Added task --> https://microbit-global.monday.com/boards/1550536443/pulses/1656945636